nengo_gui/static/components/netgraph_item.js

```diff
@@ -357,10 +358,14 @@ Nengo.NetGraphItem.prototype.generate_menu = function () {
         }
     }
     if (this.sp_targets.length > 0) {
-        items.push(['Semantic pointer cloud',
-                    function() {self.create_graph('Pointer', self.sp_targets[0]);}])
-        items.push(['Semantic pointer plot',
-            function() {self.create_graph('SpaSimilarity', self.sp_targets[0]);}])
+        if(!this.scalar_out) {
```

Comment:
I'm uncomfortable with this check.  Whatever is generating sp_targets shouldn't be giving us invalid data in the first place.